webserver/Models/DTOs/RealEstateDTO.cs

```diff
@@ -0,0 +1,34 @@
+namespace webserver.Models.DTOs;
+
+public class RealEstateDTO {
+    
+    public int Id { get; set; }
+    public string Name { get; set; } = string.Empty;
+    public string Address { get; set; } = string.Empty;
+    public decimal Price { get; set; }
+
+    public string CompanyId { get; set; } = string.Empty;
+
+    public RealEstateDTO(int id, string name, string address, decimal price, string companyId){
+
+        Id = id;
+        Name = name;
+        Address = address;
+        Price = price;
+        CompanyId = companyId;
+    }
+
+    public static explicit operator RealEstate(RealEstateDTO DTO) {
```

Comment:
Please, follow the [capitalization convention](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions).

Pascal case for identifiers, like classes, contants.
Camel case for variables.